### ISSUE TITLE
Cannot fetch/pull/push

### ISSUE THREAD

**Issue by vanthao** [gitextensions/gitextensions#5361]
**Do you want to request a *feature* or report a *bug*?**

**What is the current behavior?**
I faced an issue with git/git extensions.
I cannot fetch of pull/push 
below is the error:

remote: Not Found
fatal: repository 'https://bitbucket.org/***/' not found

Git extension: 2.51.04
git version: 2.18.0
OS: windows 10

Please help.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**

**Environment you encounter the issue:**

- GitExtensions version:
- GIT version: 
- OS version: 
- .NET version: 

**Did this work in previous version of GitExtensions (which)?**

**Comment by RussKie** [comment]
Can you do any of these operations from the command line?

**Comment by RussKie** [comment]
Have you configured remotes correctly?

**Comment by vanthao** [comment]
No, it doesn’t work for both ui and command line. It was working before, i got some glitches and then stopped working.

**Comment by vbjay** [comment]
Open git bash and run git config -l and show the result especially your
remotes setup.

**Comment by vanthao** [comment]
Hi Asbury,
Please see the results in the attachment 


Sent from Yahoo Mail for iPhone


On Wednesday, August 29, 2018, 8:50 AM, Jay Asbury <notifications@github.com> wrote:

Open git bash and run git config -l and show the result especially your
remotes setup.

**Comment by vbjay** [comment]
What attachment?
